components/ui-checkbox/stories/Interactions.tsx

```diff
@@ -0,0 +1,70 @@
+import { FC, useCallback, useMemo, useState } from 'react';
+
+import { UiCheckbox } from '~';
+
+export const Simple: FC = () => {
+  const [isChecked, setIsChecked] = useState(false);
+
+  const handleChange = useCallback((nextIsChecked: boolean) => {
+    setIsChecked(nextIsChecked);
+  }, []);
+
+  return (
+    <UiCheckbox isChecked={isChecked} onChange={handleChange} testId="target">
+      Is awesome?
+    </UiCheckbox>
+  );
+};
+
+export const Complex: FC = () => {
```

Comment:
Why do we need this story? Seems to me it should be separate component (as it is), because of we use it often

Reply:
It's like demo which reproduce checkbox list, you know, to have a live example where checkbox can have indeterminated state.

Reply:
In general cases, you can't make indeterminated state through interactions with checkbox natively.